tools/update-lib-index.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     '.vue': require('./processor')
   },
   environments: {
+    // Deprecated
```

Comment:
```suggestion
    // TODO Remove in the next major version
    /** @deprecated */
```

(and then run `npm run update`)